src/app/common/hooks/balance/use-total-balance.tsx

```diff
@@ -0,0 +1,32 @@
+import { baseCurrencyAmountInQuote } from '@app/common/money/calculate-money';
+import { i18nFormatCurrency } from '@app/common/money/format-money';
+import { useBitcoinAssetBalance } from '@app/query/bitcoin/address/address.hooks';
+import { useCryptoCurrencyMarketData } from '@app/query/common/market-data/market-data.hooks';
+import { useCurrentStacksAccountAnchoredBalances } from '@app/query/stacks/balance/balance.hooks';
+import { useCurrentBtcNativeSegwitAccountAddressIndexZero } from '@app/store/accounts/blockchain/bitcoin/native-segwit-account.hooks';
+
+export function useTotalBalance() {
+  // get market data
+  const btcMarketData = useCryptoCurrencyMarketData('BTC');
+  const stxMarketData = useCryptoCurrencyMarketData('STX');
+
+  // get stx balance
+  const { data: balances } = useCurrentStacksAccountAnchoredBalances();
+
+  // what is the difference between these two?
+  // const { data: stxUnachoredAssetBalance } = useStacksUnanchoredCryptoCurrencyAssetBalance(address);
+
+  // get btc balance
+  const btcAddress = useCurrentBtcNativeSegwitAccountAddressIndexZero();
+  const btcBalance = useBitcoinAssetBalance(btcAddress);
+
+  if (!balances || !btcBalance) return null;
+
+  // calculate total balance
+  const stxAmount = baseCurrencyAmountInQuote(balances.stx.availableStx, stxMarketData);
+  const btcAmount = baseCurrencyAmountInQuote(btcBalance.balance, btcMarketData);
+  const totalBalance = stxAmount.amount.plus(btcAmount.amount);
```

Comment:
Let's wrap in a usememo otherwise this'll be recalculated on every render. Also we should use Money type here